man/extract.Rd

```diff
@@ -135,6 +135,8 @@
 \alias{extract,nlmerMod-method}
 \alias{extract.ols}
 \alias{extract,ols-method}
+\alias{extract.panelAR}
+\alias{extract,panelAR-method}
 \alias{extract.pglm}
 \alias{extract,pglm-method}

```

Comment:
can you please insert below in the appropriate line (alphabetic order): 

 \S4method{extract}{panelAR}(model, include.rsquared = TRUE,  include.nobs = TRUE,
  include.group = TRUE, ...)